.github/workflows/packages-upload-images.yml

```
@@ -0,0 +1,224 @@
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/debs/i386/agent/**'
+              - 'packages/debs/utils/**'
+            pkg_rpm_agent_builder_amd64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/amd64/agent/**'
+              - 'packages/rpms/utils/**'
+            pkg_rpm_agent_builder_i386:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/i386/agent/**'
+              - 'packages/rpms/utils/**'
+            pkg_rpm_legacy_builder_amd64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/amd64/legacy/**'
+              - 'packages/rpms/utils/**'      
+            pkg_rpm_legacy_builder_i386:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/i386/legacy/**'
+              - 'packages/rpms/utils/**'
+            pkg_deb_agent_builder_arm64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/debs/arm64/agent/**'
+              - 'packages/debs/utils/**'
+            pkg_deb_agent_builder_armhf:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/debs/armhf/agent/**'
+              - 'packages/debs/utils/**'
+            pkg_rpm_agent_builder_arm64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/arm64/agent/**'
+              - 'packages/rpms/utils/**'
+            pkg_rpm_agent_builder_armhf:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/armhf/agent/**'
+              - 'packages/rpms/utils/**'       
+            compile_windows_agent:
+              - 'packages/windows/**'
+            commom_wpk_builder:
+              - 'packages/wpk/wpkpack.py'
+              - 'packages/wpk/run.sh'
+              - 'packages/wpk/generate_wpk_package.sh'
+              - 'packages/wpk/common/**'
+            linux_wpk_builder_x86_64: 
+              - 'packages/wpk/wpkpack.py'
+              - 'packages/wpk/run.sh'
+              - 'packages/wpk/generate_wpk_package.sh'
+              - 'packages/wpk/linux/x86_64/**'
+
+      - name: Set TAG and WAZUH_AGENT_PACKAGES_BRANCH
+        run: |
+          VERSION=$(sed 's/^v\([0-9]*\.[0-9]*\.[0-9]*\)/\1/' src/VERSION)
+          if [ "${{ inputs.docker_image_tag }}" == "auto" ]; then echo "TAG=$VERSION" >> $GITHUB_ENV;
+          elif [ "${{ inputs.docker_image_tag }}" == "developer" ]; then echo "TAG=$(sed 's|[/\]|--|g' <<< ${{ github.ref_name }})" >> $GITHUB_ENV;
+          else echo "TAG=${{ inputs.docker_image_tag }}" >> $GITHUB_ENV; fi
+          echo "WAZUH_AGENT_PACKAGES_BRANCH=master" >> $GITHUB_ENV
```

We want the `wazuh_agent_packages` branch to be the same as the one used in `wazuh`. 
As the only branches that can trigger would be `master` and a versioned branch, we can tell it to always use the `ref_name`, as it should also be in the other repository.
```suggestion
          echo "WAZUH_AGENT_PACKAGES_BRANCH=${{ github.ref_name }}" >> $GITHUB_ENV
```